ui/raidboss/data/05-shb/alliance/the_puppets_bunker.js

```diff
@@ -1,10 +1,261 @@
 'use strict';
 
-// The Puppets' Bunker
+// TODO: is it worth adding triggers for gaining/losing shield protocol? effect 8F[0-2]
 
 [{
   zoneId: ZoneId.ThePuppetsBunker,
   timelineFile: 'the_puppets_bunker.txt',
   triggers: [
+    {
+      id: 'Puppet Aegis Anti-Personnel Laser',
+      netRegex: NetRegexes.headMarker({ id: '00C6' }),
+      condition: Conditions.targetIsYou(),
+      // TODO: do we need a second "away from tank" trigger if you don't have a marker?
+      // We don't do this in Copied Factory for this same 00C6, but we could?
+      response: Responses.tankBuster('info'),
+    },
+    {
+      id: 'Puppet Aegis Beam Cannons',
+      netRegex: NetRegexes.startsUsing({ source: '813P-Operated Aegis Unit', id: '5073', capture: false }),
+      alertText: function(data) {
+        data.beamCannonCount = data.beamCannonCount || 0;
+        data.beamCannonCount++;
+
+        // TODO: 507[789A] is probably a rotation ability sync, so we could give better directions?
+        if (data.beamCannonCount <= 2) {
+          return {
+            en: 'Go To Intersection',
+          };
+        }
+        return {
+          en: 'Go To Narrow Intersection',
+        };
+      },
+    },
+    {
+      id: 'Puppet Aegis Aerial Support Surface Laser',
+      netRegex: NetRegexes.headMarker({ id: '0017' }),
+      condition: (data, matches) => data.me === matches.target && data.phase !== 'superior',
+      // TODO: should this be more along the lines of an imperative, like GTFO?
+      alarmText: {
+        en: '10x Sky Laser on YOU',
```

Comment:
It occurs to me that this is similar to the meteors on Ozma. Possibly go more prescriptive on this since that has near-universal handling? "Bait lasers left/back" is nearly short enough and captures the imperative maybe a little more?

Reply:
I think the difference from Ozma meteors is that it isn't proximity damage.  It's a chasing aoe you want to not drag through the group.  `Chasing Laser: Get Away`? I've seen two groups go to the same side, and there's plenty of room (although maybe you've seen disaster)?

Reply:
I have seen a group where two people did the sidewalk dance when they collided. Maybe not such a big deal.

I like `Chasing Laser` though. Might switch to that instead of `Sky Laser` for future work.